packages/react/src/components/DashboardEditor/DashboardEditorCardRenderer.jsx

```diff
@@ -301,7 +178,8 @@ const DashboardEditorCardRenderer = React.memo(
         onDuplicate,
         onRemove,
         isSelected,
-        onShowImageGallery
+        onShowImageGallery,
+        availableDimensions
```

Comment:
available dimensions are added, so they can be passed to the `DashboardEditorDefaultCardRender` and be used by the `BarChartCard`.